packages/atlas/src/views/viewer/VideoView/VideoView.tsx

```diff
@@ -166,61 +179,96 @@ export const VideoView: React.FC = () => {
-        ) : (
-          <SkeletonLoader height={xsMatch ? 56 : 32} width={400} />
-        )}
-        <Meta variant="t300" secondary>
-          {video ? (
-            formatVideoViewsAndDate(video.views || null, video.createdAt, { fullViews: true })
-          ) : (
-            <SkeletonLoader height={24} width={200} />
-          )}
-        </Meta>
-        <ChannelContainer>
-          <ChannelLink id={video?.channel?.id} />
-        </ChannelContainer>
-        <DescriptionContainer>
-          {video ? (
-            video.description?.split('\n').map((line, idx) => <p key={idx}>{replaceUrls(line)}</p>)
-          ) : (
-            <>
-              <DescriptionSkeletonLoader width={700} />
-              <DescriptionSkeletonLoader width={400} />
-              <DescriptionSkeletonLoader width={800} />
-              <DescriptionSkeletonLoader width={300} />
-            </>
-          )}
-        </DescriptionContainer>
-        <LicenseContainer>
-          {video ? (
-            <>
-              License:
-              {foundLicense && (
-                <a href={foundLicense.url} target="_blank" rel="noopener noreferrer">
-                  {foundLicense.name}
-                </a>
+      <StyledLimitedWidthContainer>
+        <LayoutGrid>
+          <GridItem className={transitions.names.slide} colSpan={{ xxs: 12, xs: 12, sm: 12, md: 8, xl: 8, xxl: 8 }}>
+            <TitleContainer>
+              {video ? (
+                <TitleText variant={mdMatch ? 'h600' : 'h400'}>{video.title}</TitleText>
+              ) : (
+                <SkeletonLoader height={mdMatch ? 56 : 32} width={400} />
               )}
-              <p>{video.license?.customText}</p>
-              {video.license?.attribution ? <p>Attribution: {video.license.attribution}</p> : null}
-            </>
-          ) : (
-            <SkeletonLoader height={12} width={200} />
-          )}
-        </LicenseContainer>
-        <MoreVideosContainer>
-          <InfiniteVideoGrid
-            title={`More from ${video?.channel?.title}`}
-            titleLoader
-            ready={!loading}
-            videoWhereInput={{ channel: { id_eq: channelId } }}
-            showChannel={false}
-            excludeId={video?.id}
-          />
-        </MoreVideosContainer>
-      </InfoContainer>
-    </StyledViewWrapper>
+              <Meta variant={mdMatch ? 't300' : 't100'} secondary>
+                {video ? (
+                  formatVideoViewsAndDate(video.views || null, video.createdAt, { fullViews: true })
+                ) : (
+                  <SkeletonLoader height={24} width={200} />
+                )}
+              </Meta>
+            </TitleContainer>
+            <ChannelContainer>
+              <ChannelLink followButton id={channelId} textVariant="h300" />
+            </ChannelContainer>
+            <DetailsWrapper>
+              <DescriptionContainer>
+                <DescriptionTitle variant="h100">Description</DescriptionTitle>
```

Comment:
I see the small problem here. When there is no description, you see just a title. It looks really odd on smaller screens: 
![image](https://user-images.githubusercontent.com/51168865/152743367-f238dd39-fb75-4c0e-b31f-3ad5d735aea2.png)

Could you ask the designer what we should do here? 
Maybe we shouldn't show `Description` when there is none?

Reply:
@Lenorette We need your advice here